syntax = "proto3";

message Plot {
    string name = 1; // name of image to be generated
    string parameter = 2;  // name of vehicle parameter to plot
    string colormap = 3;  // any valid matplotlib colormap
                          // (https://matplotlib.org/stable/gallery/color/colormap_reference.html)
    float subdivision = 4; // coloring step-size in meters
    float thickness = 5;
}

message VisualizationConfig {
    repeated Plot plot = 2;
}